fk/luae_na/src/main/java/se/inera/intyg/common/luae_na/v1/model/converter/RespConstants.java

```diff
@@ -21,6 +21,42 @@
 
 public class RespConstants {
 
+    public static final String DIAGNOS_CATEGORY_ID = "diagnos";
+    public static final String DIAGNOS_SVAR_ID_6 = "6";
+    public static final String DIAGNOS_BESKRIVNING_DELSVAR_ID_6 = "6.1";
+    public static final String DIAGNOS_DELSVAR_ID_6 = "6.2";
+    public static final String BIDIAGNOS_1_BESKRIVNING_DELSVAR_ID_6 = "6.3";
+    public static final String BIDIAGNOS_1_DELSVAR_ID_6 = "6.4";
+    public static final String BIDIAGNOS_2_BESKRIVNING_DELSVAR_ID_6 = "6.5";
+    public static final String BIDIAGNOS_2_DELSVAR_ID_6 = "6.6";
+    public static final String DIAGNOS_SVAR_JSON_ID_6 = "diagnoser";
+    public static final String DIAGNOS_CATEGORY_TEXT = "KAT_3.RBK";
+    public static final String DIAGNOS_SVAR_TEXT = "FRG_6.RBK";
+    public static final String DIAGNOS_SVAR_BESKRIVNING = "FRG_6.HLP";
+    public static final String DIAGNOS_DELSVAR_TEXT = "DFR_6.2.RBK";
+    public static final String DIAGNOS_ICD_10_ID = "ICD_10_SE";
+    public static final String DIAGNOS_ICD_10_LABEL = "ICD-10-SE";
+    public static final String DIAGNOS_KSH_97_ID = "KSH_97_P";
+    public static final String DIAGNOS_KSH_97_LABEL = "KSH97-P (Primärvård)";
+
+
+    public static final String DIAGNOSGRUND_SVAR_ID_7 = "7";
+    public static final String DIAGNOSGRUND_TEXT = "FRG_7.RBK";
```

Comment:
One comment for the changes... 

For naming convention add _ID to any TEXT_ID. Same for DESCRIPTION_ID. 

Name any BESKRIVNING as DESCRIPTION when it is not part of the model. 

DIAGNOS_SVAR_BESKRIVNING -> DIAGNOS_SVAR_DESCRIPTION_ID.

Reply:
Fixed!